upup/pkg/fi/cloudup/openstack/server_group.go

```diff
@@ -84,80 +84,52 @@ func listServerGroups(c OpenstackCloud, opts servergroups.ListOptsBuilder) ([]se
 	}
 }
 
-// matchInstanceGroup filters a list of instancegroups for recognized cloud groups
-func matchInstanceGroup(name string, clusterName string, instancegroups []*kops.InstanceGroup) (*kops.InstanceGroup, error) {
-	var instancegroup *kops.InstanceGroup
-	for _, g := range instancegroups {
-		var groupName string
-
-		switch g.Spec.Role {
-		case kops.InstanceGroupRoleControlPlane, kops.InstanceGroupRoleNode, kops.InstanceGroupRoleBastion:
-			groupName = clusterName + "-" + g.ObjectMeta.Name
-		default:
-			klog.Warningf("Ignoring InstanceGroup of unknown role %q", g.Spec.Role)
-			continue
-		}
-
-		if name == groupName {
-			if instancegroup != nil {
-				return nil, fmt.Errorf("found multiple instance groups matching servergrp %q", groupName)
-			}
-			instancegroup = g
-		}
-	}
-
-	return instancegroup, nil
-}
-
-func osBuildCloudInstanceGroup(c OpenstackCloud, cluster *kops.Cluster, ig *kops.InstanceGroup, g servergroups.ServerGroup, nodeMap map[string]*v1.Node) (*cloudinstances.CloudInstanceGroup, error) {
-	newLaunchConfigName := g.Name
+func osBuildCloudInstanceGroup(c OpenstackCloud, cluster *kops.Cluster, ig *kops.InstanceGroup, nodeMap map[string]*v1.Node) (*cloudinstances.CloudInstanceGroup, error) {
 	cg := &cloudinstances.CloudInstanceGroup{
-		HumanName:     newLaunchConfigName,
+		HumanName:     ig.Name,
 		InstanceGroup: ig,
 		MinSize:       int(fi.ValueOf(ig.Spec.MinSize)),
-		TargetSize:    int(fi.ValueOf(ig.Spec.MinSize)), // TODO: Retrieve the target size from OpenStack?
+		TargetSize:    int(fi.ValueOf(ig.Spec.MinSize)),
 		MaxSize:       int(fi.ValueOf(ig.Spec.MaxSize)),
-		Raw:           &g,
```

Comment:
what is purpose of this Raw field? Just thinking is it in use in some another place